doc/sphinx/proof-engine/ltac.rst

```diff
@@ -430,6 +430,19 @@ Selectors can also be used nested within a tactic expression with the
    :name: No such goal. (Goal selector)
    :undocumented:
 
+.. note:: Tactics applied to multiple goals do not necessarily preserve the order of the goals.
+   Modified goals may not appear in the order a human user may expect. (It would be great to support
```

Comment:
This could be made more precise given that the way it is *designed* today is specifically to gather all the goals in the place of the first one of the selection.

Reply:
I agree with @Zimmi48 . IMHO the manual should state the behavior of Coq as neutrally and precisely as possible.

Reply:
@Zimmi48 I don't like "designed" because all too often it is a euphemism for "that's the way it is, we don't plan to change it".  It may be possible to change the behavior as I suggested in the issue.

If multigoal constructs indeed gather the goals, I can say that.  But your statement in the issue that "multi-goal tactics may generate several goals, but we cannot associate them to any one of the selected sub-goals" suggested the opposite.  If a single goal becomes multiple goals, they are _guaranteed_ to be grouped together?

@chdoc Would be more helpful if you were more specific.